code/game/objects/structures/morgue.dm

```diff
@@ -114,7 +114,8 @@
 			connected.connected = src
 			icon_state = "morgue0"
 			for(var/atom/movable/A in src)
-				A.forceMove(connected.loc)
+				if(!isobserver(A) && A.move_resist != INFINITY)
```

Comment:
I'm pretty sure ghosts have `move_resist = INFINITY` so the first check is unneeded

Reply:
Good call, fixed